resources/hpxml.rb

```diff
@@ -90,6 +90,29 @@ def self.get_site_values(site:)
              :disable_natural_ventilation => to_bool(XMLHelper.get_value(site, "extension/DisableNaturalVentilation")) }
   end
 
+  def self.add_site_neighbor(hpxml:,
+                             azimuth:,
+                             distance:,
+                             **remainder)
+    neighbors = XMLHelper.create_elements_as_needed(hpxml, ["Building", "BuildingDetails", "BuildingSummary", "Site", "extension", "Neighbors"])
+    neighbor_building = XMLHelper.add_element(neighbors, "NeighborBuilding")
+    XMLHelper.add_element(neighbor_building, "Azimuth", to_integer(azimuth)) unless azimuth.nil?
+    XMLHelper.add_element(neighbor_building, "Distance", to_float(distance)) unless distance.nil?
```

Comment:
Remove the unless foo.nil? on these two lines. Both values are required, so if nil is provided for one of them, we'd like it cause an error here (instead of creating an HPXML file that is missing a required element).